include/f_ap/f_ap_game.h

```diff
@@ -14,6 +14,7 @@ class fapGm_HIO_c : public JORReflexible {
     virtual ~fapGm_HIO_c();
 
     void genMessage(JORMContext* ctx);
+    f32 getAspectRatio() { return mAspectRatio; }
```

Comment:
`fapGm_HIO_c::getAspectRatio()` doesn't appear in any maps for either TWW or TP.

I think you meant to create `fapGmHIO_getAspectRatio()`
![image](https://github.com/user-attachments/assets/36fd8c8b-9f01-48df-9c55-73a32dd63e52)